src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -251,7 +256,7 @@ namespace Microsoft::Console::VirtualTerminal
                                      const size_t cchBuffer,
                                      _Out_ DWORD* const pRgb);
 
-        bool _GetOscSetCursorColor(_In_reads_(cchOscString) const wchar_t* const pwchOscStringBuffer,
+        bool _GetOscSetColor(_In_reads_(cchOscString) const wchar_t* const pwchOscStringBuffer,
                                    const size_t cchOscString,

```

Comment:
nit: it looks like the indentation of the two lines following are now misaligned with the opening parenthesis